src/main/java/com/google/sps/servlets/AddToDatabaseServlet.java

```diff
@@ -0,0 +1,82 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.gson.Gson;
+import com.google.sps.data.QuestionChoice;
+import com.google.sps.data.QuestionDatabase;
+import com.google.sps.data.QuizQuestion;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.PatternSyntaxException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/addToDatabase")
+/** Responds with a JSON string containing questions and answers for the career quiz */
+public class AddToDatabaseServlet extends HttpServlet {
```

Comment:
Name this something like "StoreQuestion". The fact that it's a servlet is clear from the type and the fact that you're storing it in a database is tangential--we could replace the storage with something else.